__tests__/api.test.js

```diff
@@ -35,3 +35,25 @@ describe("/api/topics", () => {
         });
     });
 });
+
+describe("/api", () => {
+    describe("GET", () => {
+        test("200: status code of 200 is sent", () => {
```

Comment:
I wouldn't bother splitting this out, you should only really need one happy path test that checks that the endpoint works okay.